partners/scientific-communities.md

```diff
@@ -1,51 +1,65 @@
 your package will be:
 
-* Reviewed against current pyOpenSci guidelines and checks. 
+* Reviewed against current pyOpenSci guidelines and checks.
 * Also reviewed against the Pangeo-specific guidelines below.
--->
 
-<!-- 
-```{admonition} Pangeo Review Guidelines
+
+
+```{admonition} Pangeo Specific Software Peer Review Guidelines
 :class: note
 
-On top of the pyOpenSci review guidelines, we will also perform checks for 
-the following: 
+In addition to [basic pyOpenSci review checks](https://www.pyopensci.org/software-peer-review/how-to/editor-in-chief-guide.html#editor-checklist-template), when reviewing a
+Pangeo-related package, we will also perform checks for the following:
+
+* **Consume and produce high-level data structures** (e.g. **Xarray datasets**, **Pandas DataFrames**) wherever feasible:
+* **Operate lazily when called on Dask data structure:** Lazy loading refers to a tool only performing operations on data when it's needed. Lazy loading is ideal for workflows using larger datasets which will quickly consume memory and compute power when you work with them. Lazy loading allows a tool to create a connection to a dataset, but only actually access / open the part of the dataset that you require to use in your workflow.  [Learn more here.](https://docs.dask.org/en/stable/user-interfaces.html?highlight=lazy#laziness-and-computing)
 
-* **Consume and produce high-level data structures** (e.g. **Xarray datasets**, **Pandas DataFrames**) wherever feasible: 
-* **Operate lazily when called on Dask data structure:** Lazy loading refers to a tool only performing operations on data when it's needed. Lazy loading is ideal for workflows using larger datasets which will quickly consume memory and compute power when you work with them. [Learn more here.](https://saturncloud.io/blog/a-data-scientist-s-guide-to-lazy-evaluation-with-dask/)  
-TODO: Is this the best link for an overview of lazy loading - maybe there is a tutorial?? 
-* **Avoid file I/O unless your package is specifically an I/O package:** I/0 refers to data input and output of data.
-TODO: Ask tom for a better explanation of what this specific means and add a link if there is one ? 
-``` -->
+    * A few other resources related to lazy loading:
+
+        * [Dask and stages of computation](https://docs.dask.org/en/stable/phases-of-computation.html): this provides and overview of using Dask effectively to optimize your compute workflow.
+        * [Using Dask and Xarray together:](https://docs.xarray.dev/en/stable/user-guide/dask.html?highlight=lazy#using-dask-with-xarray) this page provides an overview of how to use Dask to optimize working with data arrays in Xarray.
+        * [Tutorial on using Dask and lazy loading]( http://tutorial.dask.org/01_dataframe.html )
+
+* **Avoid file input and output (I/O) unless your package is specifically an I/O package:** I/0 refers to data input and output of data.
```

Comment:
on the cloud - especially important to use cloud native formats - permissions can be challenging and it can be very slow ... 
don't write your own package for reading and writing data - use existing ones like xarray, rasterio, etc...